include/dt-bindings/gpio/gpio.h

```diff
@@ -1,180 +1,432 @@
 /*
  * Copyright (c) 2018 Linaro Limited
+ * Copyright (c) 2018 Peter Bigot Consulting, LLC
```

Comment:
Does it make sense to document the bit mask at the top of this file?  Its a bit confusing trying to parse that info out of this file.

Reply:
Yeah, I can do that, with a caveat that it's informational: if somebody changes the assigned shift for a bit or region the documentation will get out of date.

Reply:
Updated with a convenience table in the header.